apps/ai-content-generator/src/configs/features/featureConfig.tsx

```diff
@@ -30,6 +39,7 @@ const featureConfig: FeatureConfig = {
       promptHelpText: `${PROMPT_HELP_TEXT_BASE} enter a prompt to generate a title`,
       promptPlaceholder:
         "Write a prompt, for example...\nAn article about quantum computing in simple terms\nA blog post about creative ideas for a 10 year old's birthday\nA tutorial about making an HTTP request in JavaScript",
+      tokenWarning,
```

Comment:
Are there any places where the tokenWarning would/could be different? 

If not I think this could live in it's own config and get imported in the component that we're passing it into currently

Reply:
Great suggestion, that will definitely streamline this! Made the change.